src/main/java/commands/ListChapterCommand.java

```diff
@@ -0,0 +1,53 @@
+package commands;
+
+import access.Access;
+import manager.chapter.Chapter;
+import manager.module.ChapterList;
+import storage.Storage;
+import ui.Ui;
+
+import java.util.ArrayList;
+
+import static common.Messages.CHAPTER;
+
+public class ListChapterCommand extends ListCommand {
+    public static final String COMMAND_WORD = "list";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
```

Comment:
Perhaps can change chapters to %ss in ListCommand and when you use this message, Stringformat with the type Messages.CHAPTER